Enable keyboard navigation for fullscreen mode.

Saulis opened the pull request:
Fixes #61 
Fixes #62

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/vaadin/vaadin-combo-box/63)
<!-- Reviewable:end -->

jounik commented on vaadin-combo-box.html:
```diff
@@ -251,7 +251,10 @@
+      }.bind(this);
+
+      this.$.input.addEventListener('keydown', keydownHandler);
+      this.$.fullScreenInput.addEventListener('keydown', keydownHandler);
```
Would the keydown event happen to bubble up to the root element, so we could just use one event listener?

Saulis commented:
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[vaadin-combo-box.html, line 257 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/63#-K49immkHSI7tjk2LQvd-r1-257:-K4CbSc1BdrTWY3mDxLU:-1764390393) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/38aad77b948112ba8574aec68a413e5907812fa7/vaadin-combo-box.html#L257)):*
You're absolutely right, it does. I'll update the patch accordingly.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/63)*
<!-- Sent from Reviewable.io -->

tehapo commented:
Shouldn't the styles also be updated a bit? At least vaadin-combo-box.html:67 to have the focus background also on fullscreen mode.

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/63#-:-K4M8zbNOslDAQLAXrM1:311996378)*
<!-- Sent from Reviewable.io -->

Saulis commented:
Nice catch, done.

---

Review status: 1 of 2 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/63#-:-K4MQyH0t6SNqNHNmtpt:-2027112901)*
<!-- Sent from Reviewable.io -->

tehapo commented:
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[test/keyboard.html, line 29 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/63#-K4M_nl3V50MGN-6MQ--:-K4M_nl3V50MGN-6MQ-0:-1966401289) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/7c01cb947a60b7fe90c7b8eb0e9f26f8105c0811/test/keyboard.html#L29)):*
Missing a space after if. This would fail the linting step if it is merged before this one. :smile: 

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/63)*
<!-- Sent from Reviewable.io -->

Saulis commented:
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[test/keyboard.html, line 29 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/63#-K4M_nl3V50MGN-6MQ--:-K4RFDi91gT6BeuYs0KW:-2040099256) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/7c01cb947a60b7fe90c7b8eb0e9f26f8105c0811/test/keyboard.html#L29)):*
Rebased and fixed.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/63)*
<!-- Sent from Reviewable.io -->

tehapo commented:
LGTM.

---

Reviewed 2 of 2 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/63#-:-K4RIvQZzydRLYp2W8Wa:-492975211)*
<!-- Sent from Reviewable.io -->
